### PR TITLE
Please work

### PR DESCRIPTION
**Pull request by nachomdw**
Same 2 files

### CODE REVIEW

File: code/app/src/main/res/layout/entrant_home_view.xml [hunk 1]
```diff
@@ -5,78 +5,91 @@
...
         android:layout_marginEnd="16dp"
+        android:text="Upcoming Events"
+        android:textColor="#333333"
+        android:textSize="20sp"
+        android:textStyle="bold"
         app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.0"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/entrant_home_view_user_name">
+        app:layout_constraintTop_toBottomOf="@id/username" />
```
Comment (zyzCodes) [inline code/app/src/main/res/layout/entrant_home_view.xml:47]: we won't be using username, please leave this id unchanged

File: code/app/src/main/res/layout/entrant_home_view.xml [hunk 2]
```diff
@@ -5,78 +5,91 @@
...
         android:layout_marginTop="32dp"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
-        tools:srcCompat="@tools:sample/avatars" />
+        tools:srcCompat="@tools:sample/avatars[5]" />
 
+    <!-- Username Title Text -->
     <TextView
-        android:id="@+id/entrant_home_view_user_name"
```
Comment (zyzCodes) [inline code/app/src/main/res/layout/entrant_home_view.xml:19]: same here

File: code/app/src/main/res/layout/entrant_home_view.xml [hunk 3]
```diff
@@ -5,78 +5,91 @@
...
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginTop="16dp"
-        android:text="Name"
+        android:text="Username Placeholder"
+        android:textSize="18sp"
+        android:textStyle="bold"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/entrant_home_view_image_view" />
```
Comment (zyzCodes) [inline code/app/src/main/res/layout/entrant_home_view.xml:26]: please leave this id unchanged as well as it is more descriptive and easier to find

File: code/app/src/main/res/layout/entrant_home_view.xml (hunk 1 above)
Comment (zyzCodes) [reply code/app/src/main/res/layout/entrant_home_view.xml:47]: We will be using name not username due to project requirements

Comment (nachomdw) [reply code/app/src/main/res/layout/entrant_home_view.xml]: I believe I did this

File: code/app/src/main/res/layout/entrant_home_view.xml (hunk 2 above)
Comment (nachomdw) [reply code/app/src/main/res/layout/entrant_home_view.xml:19]: fixed this too!

File: code/app/src/main/res/layout/entrant_home_view.xml (hunk 3 above)
Comment (nachomdw) [reply code/app/src/main/res/layout/entrant_home_view.xml:26]: this one should also be done now as well

**Comment by nachomdw** [comment]
Okay I think that should be good to go now

**Review by zyzCodes — APPROVED** [review]

**Pull request merged**